shared/nav.android.js

```diff
@@ -91,12 +91,8 @@ class Nav extends Component {
 
   componentWillMount () {
     NativeBackAndroid.addEventListener('hardwareBackPress', () => {
-      // TODO Properly handle android back button press
-      const currentRoute = this.props.router.getIn(['tabs', this.props.router.get('activeTab'), 'uri'])
-      if (currentRoute == null || currentRoute.count() <= 1) {
-        return false
-      }
-      this.props.navigateBack()
+      // Just going up vs back for now
+      this.props.navigateUp()
```

Comment:
You should keep the currentRoute.count() logic, or check if you are at root.

Pressing back at root should quit the app, so you should `return false` if you are at the root